examples/c/test_capture_performance.c

```diff
@@ -114,14 +114,20 @@ main(int argc, char *argv[])
     printf("\nTesting frame grab performance\n");
     FILE *fp = fopen("capture_grab.csv", "w");
     assert(fp != NULL);
-    fprintf(fp, "frame,grab,retrieve,converted");
+    fprintf(fp, "frame,start,grab,retrieve,converted");
```

Comment:
Please add `start` and `diff` to the end (after all the `tracking%d` elements), so that existing code that parses the fields in order instead of reading the headers and assigning keys this way will still work (and also fix the ordering below accordingly.